modules/manage/pages/kubernetes/troubleshooting/k-troubleshoot.adoc

```diff
@@ -221,6 +221,49 @@ flux resume helmrelease <cluster-name> --namespace <namespace>
 ```
 //end::deployment-retries-exhausted[]
 
+//tag::crashloopbackoff[]
+=== Crash loop backoffs
+
+If a broker crashes after startup, and if it gets stuck in a crash loop, it could produce progressively more stored state that uses more disk space and takes more time for each restart to process.
+
+To prevent infinite crash loops, the Redpanda Helm chart sets the `crash_loop_limit` node property  to 5, which is the number of consecutive crashes that can happen within one hour of each other. After Redpanda reaches this limit, it cannot restart until its internal consecutive crash counter is reset to zero.
```

Comment:
I'm not sure we want to advertise disabling this. We explicitly set the default to 5 fairly recently in this PR: https://github.com/redpanda-data/helm-charts/pull/718

Reply:
Ooooh, interesting! Thanks for the ref. In that case we may want to add in a custom delay to the Redpanda Pods so there's a window of time to allow users to delete the file. 🤔 I'll file us an issue.

Reply:
```suggestion
To prevent infinite crash loops, the Redpanda Helm chart sets the `crash_loop_limit` node property  to 5, which is the number of consecutive crashes that can happen within one hour of each other. After Redpanda reaches this limit, it does not start until its internal consecutive crash counter is reset to zero. From the Kubernetes perspective Redpanda container is in CrashLoopBackoff.
```